src/styles/main.scss

```
@@ -1,3 +1,11 @@
+@import "utils/variables.scss";
+@import "calendar.scss";
+@import "calendar__day.scss";
```

no needs to add an extension
```suggestion
@import "utils/variables";
@import "calendar";
```